netci.groovy

```diff
@@ -69,11 +69,11 @@ def addPushJob(String project, String branch, String os, String configuration)
       def newJob = job(Utilities.getFullJobName(project, shortJobName, isPR)){
         steps{
             shell("cd ./source-build;git submodule update --init --recursive");
-            shell("cd ./source-build;./build.sh /p:ArchiveDownloadedPackages=true /p:Configuration=${configuration} ${loggingOptions}");
+            shell("cd ./source-build;./build.sh /p:ArchiveDownloadedPackages=true /p:RootRepo=known-good /p:Configuration=${configuration} ${loggingOptions}");
```

Comment:
I set the RootRepo in dir.props to known-good, so you shouldn't need to specify a RootRepo here or on the tarball build.

Reply:
Removed.